ui/component/claimPreviewTile/view.jsx

```diff
@@ -254,6 +255,8 @@ function ClaimPreviewTile(props: Props) {
         <FileThumbnail thumbnail={thumbnailUrl} allowGifs tileLayout>
           {!isChannel && (
             <React.Fragment>
+              <PreviewTilePurchaseOverlay uri={uri} />
+
```

Comment:
:small_red_triangle: This only covered tile mode. Not showing when in list-layout?